tests/test_chi_board_elections.py

```diff
@@ -2,72 +2,111 @@
 from os.path import dirname, join
 
 import pytest
-from city_scrapers_core.constants import COMMISSION, PASSED
+from city_scrapers_core.constants import COMMISSION, PASSED, TENTATIVE
 from city_scrapers_core.utils import file_response
 from freezegun import freeze_time
-from scrapy.settings import Settings
 
 from city_scrapers.spiders.chi_board_elections import ChiBoardElectionsSpider
 
 test_response = file_response(
     join(dirname(__file__), "files", "chi_board_elections.html"),
-    url="https://app.chicagoelections.com/pages/en/board-meetings.aspx",
+    url="https://chicagoelections.gov/about-board/board-meetings",
 )
 spider = ChiBoardElectionsSpider()
-spider.settings = Settings(values={"CITY_SCRAPERS_ARCHIVE": False})
 
-freezer = freeze_time("2018-11-30")
+freezer = freeze_time("2024-01-02")
 freezer.start()
 
-parsed_items = [item for item in spider._next_meeting(test_response)]
+parsed_items = [item for item in spider.parse(test_response)]
 
 freezer.stop()
 
 
+def test_meeting_count():
+    assert len(parsed_items) == 6
+
+
 def test_title():
-    assert parsed_items[0]["title"] == "Electoral Board"
+    # Note: Although this meeting is set in 2024, it appears the agency made a typo in
+    # the title. The assertion below is correct despite stating "2023".
+    assert (
+        parsed_items[0]["title"] == "Chicago Electoral Board Meeting - January 5, 2023"
+    )
+    assert (
+        parsed_items[1]["title"]
+        == "Rescheduled Regular Board Meeting - December 29, 2023"
+    )
```

Comment:
Looks like the year was fixed to 2024. Can you please update the file `tests/files/chi_board_elections.html` and update the assertions as well, and remove the `Note` comment
<img width="914" alt="Screenshot 2024-01-12 at 11 28 43" src="https://github.com/City-Bureau/city-scrapers/assets/1421838/7050c1aa-b7ae-4c9a-be20-ac354befd25e">
.